[2018-03-18 10:42:16,388][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 10:42:16,429][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiMXpe-S72mFoMfaG], source[{"_source": {"include": ["*.graphic"],"exclude": ["*.listEvent.*","*.div.desc", "*.teiHeader.*", "*.text.div" ]}}]}] on [[ssk][2]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:08:38,629][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:08:38,715][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiMXpe-S72mFoMfaG], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][2]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:23:50,537][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:23:50,560][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAia1ie-S72mFoMfb6], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][3]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:29:08,229][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:29:08,270][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAib3pe-S72mFoMfcY], source[{"_source": {"include": ["*.graphic"],"exclude": ["*.listEvent.*","*.div.desc", "*.teiHeader.*", "*.text.div" ]}}]}] on [[ssk][3]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:34:30,271][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:34:30,290][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAi7gfe-S72mFoMff0], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][0]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:43:28,894][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiiSxe-S72mFoMfc3], source[{"query":{"has_child" : {"type" : "scenario_metadata","query" : {"match_all": {}},"inner_hits" : {} }},"_source":false}"]}] on [[ssk][1]]
MapperParsingException[Cannot generate dynamic mappings of type [_source] for [_source]]
	at org.elasticsearch.index.mapper.DocumentParser.createBuilderFromFieldType(DocumentParser.java:506)
	at org.elasticsearch.index.mapper.DocumentParser.parseDynamicValue(DocumentParser.java:616)
	at org.elasticsearch.index.mapper.DocumentParser.parseValue(DocumentParser.java:444)
	at org.elasticsearch.index.mapper.DocumentParser.parseObject(DocumentParser.java:264)
	at org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:124)
	at org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:309)
	at org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:580)
	at org.elasticsearch.index.shard.IndexShard.prepareIndexOnPrimary(IndexShard.java:559)
	at org.elasticsearch.action.index.TransportIndexAction.prepareIndexOperationOnPrimary(TransportIndexAction.java:211)
	at org.elasticsearch.action.index.TransportIndexAction.executeIndexRequestOnPrimary(TransportIndexAction.java:223)
	at org.elasticsearch.action.index.TransportIndexAction.shardOperationOnPrimary(TransportIndexAction.java:157)
	at org.elasticsearch.action.index.TransportIndexAction.shardOperationOnPrimary(TransportIndexAction.java:66)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryPhase.doRun(TransportReplicationAction.java:648)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:279)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:271)
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:77)
	at org.elasticsearch.transport.TransportService$4.doRun(TransportService.java:376)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:47:31,817][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 11:47:31,823][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiRcbe-S72mFoMfa3], source[{"_source": {"include": ["*.graphic"],"exclude": ["*.listEvent.*","*.div.desc", "*.teiHeader.*", "*.text.div" ]}}]}] on [[ssk][1]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 12:35:49,629][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 12:35:49,661][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiXc0e-S72mFoMfbX], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][1]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 13:13:54,631][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 13:13:54,644][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiMXpe-S72mFoMfaG], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][2]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 13:14:27,991][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 13:14:27,996][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiiSxe-S72mFoMfc3], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][1]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 13:14:56,530][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 13:14:56,532][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAie9fe-S72mFoMfca], source[{"_source": {"include": ["*.graphic"],"exclude": ["*.listEvent.*","*.div.desc", "*.teiHeader.*", "*.text.div" ]}}]}] on [[ssk][4]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 14:21:17,321][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAimJze-S72mFoMfdQ], source[{"query":{"has_child" : {"type" : "scenario_metadata","query" : {"match_all": {}},"inner_hits" : {} }},"_source":false}"]}] on [[ssk][1]]
MapperParsingException[Cannot generate dynamic mappings of type [_source] for [_source]]
	at org.elasticsearch.index.mapper.DocumentParser.createBuilderFromFieldType(DocumentParser.java:506)
	at org.elasticsearch.index.mapper.DocumentParser.parseDynamicValue(DocumentParser.java:616)
	at org.elasticsearch.index.mapper.DocumentParser.parseValue(DocumentParser.java:444)
	at org.elasticsearch.index.mapper.DocumentParser.parseObject(DocumentParser.java:264)
	at org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:124)
	at org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:309)
	at org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:580)
	at org.elasticsearch.index.shard.IndexShard.prepareIndexOnPrimary(IndexShard.java:559)
	at org.elasticsearch.action.index.TransportIndexAction.prepareIndexOperationOnPrimary(TransportIndexAction.java:211)
	at org.elasticsearch.action.index.TransportIndexAction.executeIndexRequestOnPrimary(TransportIndexAction.java:223)
	at org.elasticsearch.action.index.TransportIndexAction.shardOperationOnPrimary(TransportIndexAction.java:157)
	at org.elasticsearch.action.index.TransportIndexAction.shardOperationOnPrimary(TransportIndexAction.java:66)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryPhase.doRun(TransportReplicationAction.java:648)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:279)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:271)
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:77)
	at org.elasticsearch.transport.TransportService$4.doRun(TransportService.java:376)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 15:57:33,180][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 15:57:33,243][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAia1ie-S72mFoMfb6], source[{"_source": {"include": ["*.graphic"],"exclude": ["*.listEvent.*","*.div.desc", "*.teiHeader.*", "*.text.div" ]}}]}] on [[ssk][3]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 16:10:17,164][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 16:10:17,212][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAia1ie-S72mFoMfb6], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][3]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 20:14:23,185][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 20:14:23,276][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiXc0e-S72mFoMfbX], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][1]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 21:51:02,759][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 21:51:02,888][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAiXc0e-S72mFoMfbX], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][1]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 21:51:12,003][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 21:51:12,006][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAixXte-S72mFoMfed], source[{"_source": {"include": ["*.graphic"],"exclude": ["*.listEvent.*","*.div.desc", "*.teiHeader.*", "*.text.div" ]}}]}] on [[ssk][2]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 22:55:16,991][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 22:55:17,225][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAi7gfe-S72mFoMff0], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][0]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 23:52:23,353][DEBUG][action.admin.indices.mapping.put] [Malice] failed to put mappings on indices [[ssk]], type [scenario]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2018-03-18 23:52:23,423][DEBUG][action.index             ] [Malice] failed to execute [index {[ssk][scenario][AWIAisPFe-S72mFoMfd7], source[{"_source": {"include": ["*.desc.lang","*.desc.content"],"exclude": ["*.listEvent.*","*.text.div","*.desc.term", "*.desc.type", "*.teiHeader.*" ]}}]}] on [[ssk][1]]
java.lang.IllegalArgumentException: Field [_source] is defined both as an object and a field in [scenario]
	at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:494)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:376)
	at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:320)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.applyRequest(MetaDataMappingService.java:306)
	at org.elasticsearch.cluster.metadata.MetaDataMappingService$PutMappingExecutor.execute(MetaDataMappingService.java:230)
	at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
	at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
	at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
